packages/core/package.json

```
@@ -90,6 +93,8 @@
     "execa": "^5.1.1",
     "express": "^4.17.1",
     "fast-deep-equal": "^3.1.3",
+    "fast-glob": "^3.2.7",
+    "file-type": "^16.5.3",
```

```suggestion
-    "file-type": "^16.5.3",
```

Looks like file-type isn't being used anywhere in the codebase